### ISSUE TITLE
Blob txs pending on 4844-devnet-5

### ISSUE THREAD

**Issue by syntrust** [Inphi/blob-utils#9]
Hello, 
I'm using blob-utils to submit blob tx to https://rpc.4844-devnet-5.ethpandaops.io. It was submitted successfully but the tx is always pending. 
Can you help?

**Comment by syntrust** [comment]
Perhaps another issue, but when I download blob using:
```
 ./blob-utils download  --beacon-p2p-addr /ip4/127.0.0.1/tcp/9000 --slot 62647
2023/05/09 08:21:44 App failed: unsupported for the latest spec
```
Need upgrade?

**Comment by Inphi** [comment]
> Hello, I'm using blob-utils to submit blob tx to https://rpc.4844-devnet-5.ethpandaops.io. It was submitted successfully but the tx is always pending. Can you help?

You should try again. The https://rpc.4844-devnet-5.ethpandaops.io/ endpoint was out of sync yesterday but it up to date today.



> Perhaps another issue, but when I download blob using:
> 
> ```
>  ./blob-utils download  --beacon-p2p-addr /ip4/127.0.0.1/tcp/9000 --slot 62647
> 2023/05/09 08:21:44 App failed: unsupported for the latest spec
> ```
> 
> Need upgrade?

Sorry I should make the error clearer here. Downloading blobs is no longer supported with blob-utils. There have been major breaking changes to the way blobs work in the CL and I haven't updated blob-utils to support that.

**Comment by syntrust** [comment]
@Inphi  Thanks a lot!
